File: app/src/main/java/com/zivapp/notes/network/NetworkOfflineSecurity.java
package com.zivapp.notes.network;

import android.app.Application;

import com.google.firebase.database.FirebaseDatabase;

/**
 * Saving data on local database if offline
 */
public class NetworkOfflineSecurity extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        FirebaseDatabase.getInstance().setPersistenceEnabled(true);
    }
}

File: Kotlin/app/src/main/java/com/zivapp/countandnote/network/NetworkOfflineSecurity.kt
package com.zivapp.countandnote.network

import android.app.Application
import com.google.firebase.database.FirebaseDatabase

/**
 * Saving data on local database if offline
 */
class NetworkOfflineSecurity : Application() {
    override fun onCreate() {
        super.onCreate()
        FirebaseDatabase.getInstance().setPersistenceEnabled(true)
    }
}